starter/source/materials/mat/mat070/law70_upd.F

```diff
@@ -310,10 +310,10 @@ SUBROUTINE LAW70_UPD(MAT_PARAM,TITR     ,MAT_ID   ,NUPARAM  ,UPARAM   ,
 c------------------------------------
 c     static function value at EPSMAX
 c------------------------------------
-      EPSMAX = UPARAM(4)
       K = 1
       DO WHILE (MAT_PARAM%TABLE(LOAD)%X(1)%VALUES(K) < EPSMAX)
         K = K + 1
+        IF (K == LEN(1)-1) EXIT
```

Comment:
Hi @MaciekWronski 

can you double check this ? can you explain why LEN(1)-1 is the criteria to exit ?
( as a remark it is recommended to avoid using EXIT )

Thanks,
Sébastien

Reply:
I have changed the loop formulation to avoid "EXIT" , but LEN(1)-1 criterion is correct. I need to stop before the last available point because next I need to calculate the derivative using (k+1) index. Of course it could have been possible to code it in another way but it's an arbitrary choice.